docs/logging-usage.rst

```diff
@@ -392,70 +392,105 @@ Delete a sink:
 Integration with Python logging module
 ---------------------------------------------
 
-
-It's possible to tie the Python :mod:`logging` module directly into Google Cloud Logging. To use it,
-create a :class:`CloudLoggingHandler <google.cloud.logging.CloudLoggingHandler>` instance from your
-Logging client.
+It's possible to tie the Python :mod:`logging` module directly into Google Stackdriver Logging.
+There are different handler options to accomplish this. To automatically pick the default,
```

Comment:
1. Can you obey the 80 char limit in RST files too (if for no other reason than making it easy to comment on them in code reviews)
1. Suggest: "pick the default **for your current environment**,"